stores/ContractStore.ts

```diff
@@ -1,7 +1,15 @@
 import { Store } from "pullstate";
-import { Dripper, Ogv, Ousd, Vault, Veogv } from "../types/contracts";
+import {
+  Dripper,
+  Oeth_vault,
+  Ogv,
+  Ousd,
+  Vault,
+  Veogv,
+} from "../types/contracts";
 
 interface Contracts {
+  oeth: Oeth_vault;
```

Comment:
Would be better to name it `oethVault`, it's confusing rn since `OETH` can refer to the token

Reply:
Yup... makes sense. Will make this change.